test/pagespeed_test.conf.template

```diff
@@ -1905,7 +1905,9 @@ http {
     }
 
     location /mod_pagespeed_test/ipro/instant/deadline/ {
-      pagespeed InPlaceRewriteDeadlineMs -1;
+      set $ps_deadline -1;
```

Comment:
Can you add a test dependency on one of the ones that takes more than one argument and is parsed manually?